### ISSUE TITLE
Getting - Try signing in with a different account.

### ISSUE THREAD

**Issue by ats1999** [nextauthjs/next-auth#3241]
### Description 🐜

I am trying to sign with google provider, but I am getting a message `Try signing in with a different account.`
![Screenshot from 2021-11-20 07-51-11](https://user-images.githubusercontent.com/54087826/142711167-d2ec30ae-8ca0-484b-87df-19da727186b3.png)

The debug option is also does not log anything

### Is this a bug in your own project?

No

### How to reproduce ☕️

The repo is open source - https://github.com/ats1999/time-stamp

### Environment 🖥

OS - Ubuntu  20.04 LTS
Browser - Chrome Latest
```j
npx: installed 1 in 2.315s

  System:
    OS: Linux 5.11 Ubuntu 20.04.3 LTS (Focal Fossa)
    CPU: (2) x64 AMD A6-9225 RADEON R4, 5 COMPUTE CORES 2C+3G
    Memory: 633.18 MB / 6.68 GB
    Container: Yes
    Shell: 5.0.17 - /bin/bash
  Binaries:
    Node: 14.17.6 - /usr/local/bin/node
    npm: 6.14.15 - /usr/local/bin/npm
  Browsers:
    Brave Browser: 96.1.32.106
    Chrome: 96.0.4664.45
  npmPackages:
    next: ^12.0.4 => 12.0.4 
    next-auth: ^4.0.0-beta.7 => 4.0.0-beta.7 
    react: ^17.0.2 => 17.0.2 

```

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
So to reproduce, you just simply trying to log in with Google, and there is no error in the console/terminal? 🤔

**Comment by ats1999** [comment]
> 

Yes, there is no error into console or terminal

**Comment by balazsorban44** [comment]
ok, strange, but I will check. what about the two other providers?

**Comment by ats1999** [comment]
I tried with three accounts

**Comment by balazsorban44** [comment]
so all three fails? 🤔

**Comment by ats1999** [comment]
Sorry for the posing issue, there was an issue in `.env` file

**Comment by balazsorban44** [comment]
it was my next question... 😂 nice! 🎉

**Comment by varora1406** [comment]
I would like this ticket to be re-opened.

I am facing this issue. Repo - https://github.com/VB-Inc/riseInSocial
------
I checked the Twitter Developer account and found `3-legged OAuth` was not enabled.
Enabled that as well with valid deployed site details.
Even checked app with `debug` true

After multiple tries including many other things, I am not able to figure out why it's not working.
I am sure it has something with Twitter API keys, as I tried `next-auth-example` locally and it gave the same issue.

---

I am looking for help in solution, also looking to improve documentation on Twitter Developer App settings
